app-backend/db/src/main/scala/CampaignDao.scala

```diff
@@ -262,6 +262,7 @@ object CampaignDao extends Dao[Campaign] with ObjectPermissions[Campaign] {
         AND task_type = ${TaskType.Review.toString}::task_type
         AND annotation_projects.campaign_id IN (select id from candidate_campaigns)
         AND (locked_by = ${user.id} OR locked_by IS NULL)
+        AND status = ${TaskStatus.Labeled.toString}::task_status
```

Comment:
should it be... _only_ labeled? what if a user has a locked task in validation in progress? they'll never get that task again this way

Reply:
I think that's OK as it would be unlocked and taken by someone else if they abandon it. Does that seem right to you?

Reply:
Yeah it seems fine -- it adds some lag before a task can be picked up by someone else, which might be a drag if the bulk of labeling occurs in a classroom setting / all at the same time, but maybe that won't be an issue.